feedback_tool.py

```diff
@@ -0,0 +1,71 @@
+import psycopg2
+import random
+import streamlit as st
+from transformers import BartForConditionalGeneration, BartTokenizer
+
+# establishing connection to the database used
+my_connection = psycopg2.connect(database="postgres", user="souadyakubu", password="souadseny", host="localhost", port="5432")
```

Comment:
We definitely don't want secrets in our git repo. Alternative: https://docs.streamlit.io/library/advanced-features/secrets-management